developer/src/kmc-package/test/test-package-compiler.ts

```diff
@@ -61,26 +91,27 @@ describe('KmpCompiler', function () {
 
     let kmpJson = null;
     assert.doesNotThrow(() => {
-      kmpJson = kmpCompiler.transformKpsToKmpObject(kpsPath, source);
+      kmpJson = kmpCompiler.transformKpsToKmpObject(source, kpsPath);
     });
 
     const kmpData = await kmpCompiler.buildKmpFile(kpsPath, kmpJson);
 
     let jszip = await zip.loadAsync(kmpData);
+    assert.isNotNull(jszip.file('kmp.json')); // kmp.json should be present
     // kmp file should contain the following files
     const expectedFiles = [
       'FONTLOG.txt', 'image002.png', 'KAK_Documentation_EN.pdf', 'KAK_Documentation_KH.pdf',
       'keyboard_layout.png', 'khmer_angkor.js', 'khmer_angkor.kmx', 'khmer_angkor.kvk',
       'khmer_busra_kbd.ttf', 'Mondulkiri-R.ttf', 'OFL.txt', 'OFL-FAQ.txt', 'readme.htm',
-      'splash.gif', 'welcome.htm'
+      'splash.gif', 'welcome.htm',
+      'kmp.json', // this seems to be included as well.
+      'withfolders.qaa.sencoten.model.js', // TODO-LDML ??!!! why is this included here?
+      'example.qaa.sencoten.model.js', // TODO-LDML ??!!! why is this included here?
```

Comment:
@mcdurdin ⚠️  very puzzled as to why these are here. Obviously some kind of leftover data, something not getting cleared.

Reply:
If we don't fix now, we need to open a blocking issue to address this ... it's weird.

Reply:
Sounds like there's a logic issue with kmp.json anyway.

Reply:
solved now. JSZip was at the wrong scope (in a 3 year old commit)

Reply:
And I assume this was picked up because we merged two sets of unit tests which triggered the problem?

Reply:
I assume so. In normal use the compiler's run one time per process (node invocation) so I don't think it's hit. It's still a little puzzling. But in the end I suspect it's either a latent bug, or a change in JSZip, or both.

Reply:
I'm going with 'latent bug'.